integration-tests/tests/src/tests/sync/upload-delete-download.ts

```diff
@@ -18,7 +18,7 @@
 
 import { closeAndReopenRealm } from "../../utils/close-realm";
 
-export function itUploadsDeletesAndDownloads(): void {
+export function itUploadsDeletesAndDownloads(flexibleSync = false): void {
```

Comment:
I believe it would be more clean if this was read from the `realm.syncSession.config` instead of getting passed as an argument.

Reply:
Makes sense!